app/main.js

```diff
@@ -450,10 +455,15 @@ function startMicrobreak () {
     microbreakWinLocal.setSize(windowOptions.width, windowOptions.height)
     // microbreakWinLocal.webContents.openDevTools()
     microbreakWinLocal.once('ready-to-show', () => {
-      microbreakWinLocal.showInactive()
+      if (showBreaksAsRegularWindows) {
+        microbreakWinLocal.show()
+      } else {
+        microbreakWinLocal.showInactive()
+      }
+
       log.info(`Stretchly: showing window ${localDisplayId + 1} of ${numberOfDisplays()}`)
       if (process.platform === 'darwin') {
```

Comment:
Did you test on macOS, then? Elsewhere you wrote you can't test there

Reply:
No, I did not. Sadly, I don't have access to a macOS device.

Reply:
Fullscreen does not seems to work, will try investigate later

Reply:
I think it should be 
```
      if (process.platform === 'darwin') {
        if (showBreaksAsRegularWindows) {
          microbreakWinLocal.setFullScreen(settings.get('fullscreen'))          
        } else {
          microbreakWinLocal.setKiosk(settings.get('fullscreen'))
        }
      }
```

Reply:
Yup! That makes a lot of sense to me - I've applied those changes here - https://github.com/hovancik/stretchly/pull/875/commits/a66893e72dc6d7115bfebb52f0ab4b4f8a5e4db0